app/src/main/java/com/flatstack/android/DI.java

```diff
@@ -44,14 +47,14 @@ public static OkHttpClient getHttpClient() {
 
     private static volatile Picasso picassoInstance = null;
 
-    public static Picasso getPicasso() {
+    public static @NonNull Picasso getPicasso(@NonNull Context context) {
         Picasso localInstance = picassoInstance;
```

Comment:
What if Picasso will be called with ContextWrapper from `View`? Then `OkHttpClient` and `Picasso` singletons will be created with it. Any consequences?